include/dali/core/common.h

```diff
@@ -111,6 +111,12 @@ enum DALIImageType {
   DALI_ANY_DATA     = 4
 };
 
+enum class DALIExtNoCopyMode {
```

Comment:
I wonder if it deserves to be placed in common.h.

Reply:
I can move it somewhere to external source or pipeline.h

Reply:
Maybe external_source.h. Pipeline.h includes it anyway.

Reply:
Done.